build/push-image.sh

```diff
@@ -42,5 +42,13 @@ echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
 docker push cortexlabs/${image}:${CORTEX_VERSION}
 
 if [ "$slim" == "true" ]; then
-  docker push cortexlabs/${image}-slim:${CORTEX_VERSION}
+  if [ "$image" == "python-predictor-gpu" ]; then
+    cuda=("10.0" "10.1" "10.2" "11.0")
+
+    for i in ${!cuda[@]}; do
+      docker push cortexlabs/${image}-slim:${CORTEX_VERSION}-cuda${cuda[$i]}
+    done
```

Comment:
Another options is to do something like this, feel free to choose whichever you prefer:

```bash
for cuda in 10.0 10.1 10.2 11.0; do
  docker push cortexlabs/${image}-slim:${CORTEX_VERSION}-cuda${cuda}
done
```

Reply:
Yep, I like this one too. I went with it.